src/installer.ts

```diff
@@ -17,6 +16,11 @@ export interface DotnetVersion {
   qualityFlag: boolean;
 }
 
+enum DotnetInstallerLimits {
```

Comment:
I would use just simple constants here. Enum seems a bit excessive for such purpose because it won't be used as type. Also, enums in general have somewhat unique behavior in Typescript which may cause confusions sometimes. 

```typescript
const QUALITY_INPUT_MINIMAL_MAJOR_TAG = 6;
const LATEST_PATCH_SYNTAX_MINIMAL_MAJOR_TAG = 5;
```

that's not critical though

Reply:
@nikolai-laevskii Thanks for the comment! Fixed.